tools/container-registry/doc.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ionos-cloud/ionosctl/v6/commands"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/core"
+)
+
+func main() {
```

Comment:
There were some changes to master which allow you to reduce the code in this `doc.go` file

https://github.com/ionos-cloud/ionosctl/blob/4574e05b81b61b5e49eef97dcc2865ddd3dbcc59/tools/dataplatform/doc.go